### PR TITLE
feat: add hint around text editing

### PR DESCRIPTION
**Pull request by dwelle**
It's not obvious (https://github.com/excalidraw/excalidraw/issues/3707) since we don't state it anywhere.

Added hints:

- `enter/double-click to edit`
	- [x] Should we add it to the help dialog as well?
- `esc/ctrl-enter to finish editing`

- [ ] should we have different hints for mobile/desktop?

### CODE REVIEW

**Comment by h7y** [comment]
> - `enter/double-click to edit`
> 	- [ ] Should we add it to the help dialog as well?

Makes sense to add it to the help dialog. Hints can be missed (I personally haven't noticed hints much)

File: src/locales/en.json [hunk 1]
```diff
@@ -180,6 +180,8 @@
     "linearElement": "Click to start multiple points, drag for single line",
     "freeDraw": "Click and drag, release when you're finished",
     "text": "Tip: you can also add text by double-clicking anywhere with the selection tool",
+    "text_selected": "Double-click or press ENTER to edit text",
+    "text_editing": "Press Escape or CtrlOrCmd+ENTER to finish editing",
```
[the remark this answers is not in the document]
Comment (dwelle) [reply src/locales/en.json:184]: It's meant for logical grouping which I have noticed we're doing elsewhere in the file. Questions is whether it should be grouped or not.

[the remark this answers is not in the document]
Comment (h7y) [reply src/locales/en.json:184]: Oh got it! Then it seems fine 👍. We can probably document that in CONTRIBUTING.md :)

**Review by h7y — APPROVED** [review]

**Review by ad1992 — APPROVED** [review]
> should we have different hints for mobile/desktop?
Not needed IMO, we can merge this and improve later.

**Pull request merged**